metrics/validate_drp.yaml

```diff
@@ -536,6 +536,28 @@ TE2:
     must have median absolute value less than this for separations > 5 arcmin.
   tags: image_quality
 
+TE3:
+  reference:
+    doc: LPM-17
+    url: http://ls.st/lpm-17
+    page: 38
+  unit: ''
+  description: >
+    The averaged E1, E2, and Ex residual PSF ellipticity correlations from visits
+    must have median absolute value less than this for separations < 5 arcmin.
+  tags: image_quality
+
+TE4:
+  reference:
+    doc: LPM-17
+    url: http://ls.st/lpm-17
+    page: 31
```

Comment:
FWIW, it looks like the reference page for TE1, TE2, TE3 and TE4 should all be 38.  I don't know why TE3 is the only one that is right.

Reply:
Oops - TE3 is right because I just updated that. But then I forgot to update the others. Will do so now.